src/main/java/com/eighthlight/SocketWrapperSpy.java

```diff
@@ -0,0 +1,56 @@
+package com.eighthlight;
+
+import java.io.IOException;
+import java.io.*;
+
+public class SocketWrapperSpy implements SocketInterface {
+    private boolean createdAndListened = false;
+    private boolean stopped = false;
+    private BufferedReader in;
+//    private PrintWriter out;
```

Comment:
I assume you didn't mean to keep this here?

Reply:
oops!😅